.gitignore

```diff
@@ -4,7 +4,9 @@
 /dist
 /tmp
 /out-tsc
-/release-builds
+/app/mac
+/app/windows
+/app/linux
```

Comment:
Why not ignore `/app`, instead of ignoring each of its subdirectories? Are there files in `/app` that will be checked in?